src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchDAOQuerySupport.scala

```diff
@@ -94,7 +95,7 @@ trait ElasticSearchDAOQuerySupport extends ElasticSearchDAOSupport {
       query.must(fieldQuery)
     }
     criteria.researchPurpose map { rp =>
-      query.must(researchPurposeDAO.researchPurposeFilters(rp))
+      query.must(researchPurposeSupport.researchPurposeFilters(rp, Some(s"${AttributeName.libraryNamespace}${AttributeName.delimiter}")))
```

Comment:
Not thrilled that this prefix building logic is outside of `AttributeName`. May still work on improving this.